radsat-sk/operation/crypt/decrypt.c

```diff
@@ -0,0 +1,21 @@
+#include "decrypt.h"
+
+
+size_t decrypt(uint8_t* ciphertext, int ciphertextlen);
+
+/**
+ * @brief Encrypts a buffer in with an AES CBC cipher
+ * @header "comms-processing/crypt/encrypt.h"
+ * @param plaintext - plaintext value to be decrypt
+ * @param plaintextLen - The length of the plaintext value
+ * @return The actual size of the data
+ */
+size_t decrypt(uint8_t* ciphertext, int ciphertextlen){
```

Comment:
decrypt might be a commonly used term - and functions have global scope (unless prepended with the `static` keyword). I recommend naming it "aesDecrypt" or something. Also, are there more descriptive argument names possible?